cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -3896,27 +3896,27 @@ fn test_x64_emit() {
     // XMM_CMP_RM_R
 
     insns.push((
-        Inst::xmm_cmp_rm_r(SseOpcode::Ucomiss, RegMem::reg(xmm1), xmm2),
+        Inst::xmm_cmp_rm_r(SseOpcode::Ucomiss, xmm2, RegMem::reg(xmm1)),
         "0F2ED1",
-        "ucomiss %xmm1, %xmm2",
+        "ucomiss %xmm2, %xmm1",
```

Comment:
The changes to the disassembly order are a little confusing to me here since we're still retaining AT&T-style percent-register names (and pretty-printing this alongside other AT&T-syntax insts). At one point the text was even assemblable with GNU as (though that's almost certainly not the case anymore due to pseudoinsts!).

So even though we're swapping the args in the Rust enum and the ISLE-level helper, I think we should probably keep unchanged pretty-printing output here?

Reply:
Sure yeah, I'll note though that in my opinion there's not a ton of consistency right now. Most instructions are printed as a blend of att/intel syntax:

* Operands are left-to-right and ordered first. The left-to-right ness is Intel and the ordered first is att.
* Results are last, which is att-style